samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -161,32 +187,28 @@ The user interface of the sample depends on the hardware platform you are using.
          * Blinks at a 1 second interval if the recovery mode is active on the device and the identification mode is inactive on the device.
 
       Button 1:
```

Comment:
If this Button 1 section seems overloaded, we move a significant portion of the content to the "Fast Pair advertising policy" section and refer to it in the Button 1 description. This approach seems like a good direction since we have to variants of this information